### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by Jayhot77** [w-okada/voice-changer#714] [General]
I have a problem with the voice changer, its running good but it doesnt change my voice. I dont know if i missed something but i followed 2 tutorials and i just hear myself like normal. Please Help

![image](https://github.com/w-okada/voice-changer/assets/142265669/e072c771-1ac7-4d6e-9ae9-5f12b593907f)

**Comment by Bastianx2012** [comment]
I dont have an answer but seeing as how u are on a new version, how do I update? I tried looking everywhere

**Comment by joshtori** [comment]
I've had similar problem with the latest update, everything works fine but it doesn't change my voice, every troubleshooting video on youtube refers to the older version and don't quite help, Anyone with a solution??
![screenshot](https://github.com/w-okada/voice-changer/assets/142396038/7e9c5086-a45a-4784-a047-ec6b0b1b9e4f)

**Comment by FernThatCodes** [comment]
You need to set a device such as a speak or headphones in the "output" section under audio. @Jayhot77 
<img width="317" alt="image" src="https://github.com/w-okada/voice-changer/assets/139295348/6f1f4912-4221-4a06-8700-031e84c6ff15">

**Comment by FernThatCodes** [comment]
You can also try tweaking the tune section as It changes the pitch of your voice
